src/app/add-employee/add-employee.component.ts

```diff
@@ -1,17 +1,16 @@
 import { CommonModule } from '@angular/common';
 import { Component, Inject } from '@angular/core';
-import { AbstractControl, FormControl, FormGroup, ReactiveFormsModule, ValidationErrors, ValidatorFn, Validators } from '@angular/forms';
+import { FormControl, FormGroup, ReactiveFormsModule, Validators } from '@angular/forms';
 import { MatFormFieldModule } from '@angular/material/form-field';
 import { MatIconModule } from '@angular/material/icon';
 import { MatInputModule } from '@angular/material/input';
 import { MatDatepickerModule } from '@angular/material/datepicker';
-import { MatNativeDateModule } from '@angular/material/core';
 import { BusinessService } from '../services/business.service';
 import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
 import { IBusiness, IEmployee } from '../services/business.interface';
 import { MatButtonModule } from '@angular/material/button';
 import { MatDateFnsModule } from '@angular/material-date-fns-adapter';
-import { parse, isValid } from 'date-fns';
+import { CustomValidators } from '../validators/date-validator';
 
 @Component({
```

Comment:
Why are all the imports inside the component and not inside the module itself

Reply:
I am using standalone components which allow you to import modules inside of components, without creating module itself.